composer.json

```diff
@@ -101,7 +101,8 @@
     },
     "require-dev": {
         "grasmash/drupal-security-warning": "^1.1",
-        "phing/phing": "^2.16"
+        "phing/phing": "^2.16",
+        "zaporylie/composer-drupal-optimizations": "^1.1"
```

Comment:
@simesy  - any idea if there is a benefit/downside to this being in require-dev - all other examples I've seen have it in require to ensure it's called every time?

Reply:
yes I think it needs to be in require otherwise it won't always applly.